src/main/java/Measurement.java

```diff
@@ -1,19 +1,47 @@
 import org.json.JSONArray;
 import org.json.JSONObject;
 import java.util.ArrayList;
+import java.util.Date;
 import java.util.List;
-
+//this class will hold jobs
 public class Measurement{
-        static List<JSONObject> activeTask=new ArrayList<JSONObject>();
+   private static List<Job> activeJobs=new ArrayList<Job>();
```

Comment:
Are we going to be changing this to a priority queue?

Reply:
Probably was just thinking of a way to limit the number of jobs a phone received from the orch as its getting all of them now, so i guess the higher priority ones should be sent first